docs/Using Randomness/get-a-random-number.md

```diff
@@ -111,6 +111,9 @@ The deployed contract requests random values from Chainlink VRF, receives those
 
 You deployed a simple contract that can request and receive random values from Chainlink VRF. To see more advanced examples where the contract can complete the entire process including subscription setup and management, see the [Example Contracts](/docs/chainlink-vrf/example-contracts/) page.
 
+> 📘 Note on Requesting Randomness
```

Comment:
Which issue is this PR resolving? Why is this security consideration worth the extra highlight here?